app/views/add_menu.py

```diff
@@ -20,13 +22,14 @@ def post(self):
                     try:
                         menu.add_menu_item()
                         if MenuDbQueries().fetch_menu(menu):
-                            return{"response":"menu successfuly created"}
-                        return {"response":"unsuccessful, server error"}
+                            print(MenuDbQueries().fetch_menu(menu))
```

Comment:
@kafuuma .Do not leave print statements in your code base

Reply:
let remove them